<?xml version="1.0" encoding="utf-8"?>
<TestRun id="522d3475-6993-42b3-a934-144068790442" name="anna@cloudtest-win10 2023-01-22 23:29:16" runUser="cloudtest-win10\anna" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2023-01-22T23:29:16.6265688+00:00" queuing="2023-01-22T23:29:16.6265700+00:00" start="2023-01-22T23:27:51.8302065+00:00" finish="2023-01-22T23:29:16.6641742+00:00" />
  <TestSettings name="default" id="d1d2ec6b-2118-4d82-8cff-69989ba74607">
    <Deployment runDeploymentRoot="anna_cloudtest-win10_2023-01-22_23_29_16" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="d38be66b-b850-4e51-a550-7bb60652d796" testId="b8585ed4-c43f-c998-ca87-19216a098e70" testName="Tester.AzureUtils.AzureTableDataManagerStressTests.AzureTableDataManagerStressTests_WriteAlot_MultiPartition" computerName="cloudtest-win10" duration="00:01:19.9266303" startTime="2023-01-22T23:29:16.3115120+00:00" endTime="2023-01-22T23:29:16.3115578+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="d38be66b-b850-4e51-a550-7bb60652d796">
      <Output>
        <StdOut>Iterations=2000, Batch=1000, Partitions=100&#xD;
AzureTableDataManagerStressTests_WriteAlot_MultiPartition has written 1000 rows in 00:00:38.9284113 at 25.688178928133247 RPS&#xD;
AzureTableDataManagerStressTests_WriteAlot_MultiPartition completed. Wrote 2000 entries to 100 partition(s) in 00:01:19.2405700 at 25.239596333040005 RPS</StdOut>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="Tester.AzureUtils.AzureTableDataManagerStressTests.AzureTableDataManagerStressTests_WriteAlot_MultiPartition" storage="c:\users\anna\orleans\test\extensions\testerazureutils\bin\debug\net7.0\tester.azureutils.dll" id="b8585ed4-c43f-c998-ca87-19216a098e70">
      <Execution id="d38be66b-b850-4e51-a550-7bb60652d796" />
      <TestMethod codeBase="C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\bin\Debug\net7.0\Tester.AzureUtils.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="Tester.AzureUtils.AzureTableDataManagerStressTests" name="AzureTableDataManagerStressTests_WriteAlot_MultiPartition" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="b8585ed4-c43f-c998-ca87-19216a098e70" executionId="d38be66b-b850-4e51-a550-7bb60652d796" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Completed">
    <Counters total="1" executed="1" passed="1" failed="0" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.5+1caef2f33e (64-bit .NET 7.0.2)&#xD;
[xUnit.net 00:00:01.28]   Discovering: Tester.AzureUtils (method display = ClassAndMethod, method display options = None)&#xD;
[xUnit.net 00:00:01.49]   Discovered:  Tester.AzureUtils (found 246 test cases)&#xD;
[xUnit.net 00:00:01.50]   Starting:    Tester.AzureUtils (parallel test collections = off, max threads = 4)&#xD;
[xUnit.net 00:01:21.54]   Finished:    Tester.AzureUtils&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="cloudtest-win10" outcome="Warning" timestamp="2023-01-22T23:29:16.4757610+00:00">
        <Text>Data collector 'Blame' message: All tests finished running, Sequence file will not be generated.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>